app/models/read_text.rb

```diff
@@ -0,0 +1,5 @@
+class ReadText < ApplicationRecord
+  validates :user_id, uniqueness: {
+              scope: :movie_id,
+            }
```

Comment:
`belongs_to` が抜けています。

```suggestion
  belongs_to :user
  belongs_to :aws_text
  validates :user_id, uniqueness: {
              scope: :movie_id,
            }
```